app/src/main/java/org/thoughtcrime/securesms/database/ThreadDatabase.java

```diff
@@ -771,11 +773,11 @@ record = reader.getNext();
           deleteThread(threadId);
           return true;
         }
+        // todo: add empty snippet that clears existing data
         return false;
       }
     } finally {
-      if (reader != null)
-        reader.close();
+      if (reader != null) { reader.close(); }
```

Comment:
if we're changing it, consider try-with resources.

Reply:
Converted to try-with-resources